physionet-django/console/services.py

```diff
@@ -0,0 +1,34 @@
+import logging
+import re
+from typing import Optional
+
+from pdfminer.high_level import extract_text
+
+from user.models import Training
+
+logging.getLogger("pdfminer").setLevel(logging.WARNING)
+
+
+def _get_regex_value_from_text(text: str, regex: str) -> str:
```

Comment:
Clearly the return type is wrong - this function looks like it could return a None.